features/supplier/supplier_account.feature

```diff
@@ -28,7 +28,7 @@ Scenario: Supplier user can provide and change supplier details before confirmin
   Then I am on the 'Registered company name' page
   And I enter 'Toys "ᴙ" Us' in the 'Registered company name' field and click its associated 'Save and return' button
 
-  Then I click the summary table 'Answer required' link for 'Registered company address'
+  Then I click the summary table 'Change' link for 'Registered company address'
```

Comment:
Why is this change necessary? 🤔

Reply:
Because the test was previously expecting this field to be blank (having not been initialized by `create_supplier` and so showing `Answer required` instead of `Change` as the link.

Reply:
That's really weird, how was it passing before?

Reply:
It had clearly had its supplier-level address copied across from a declaration at some point. Someone testing a script?